phasmophobia_evidence_v3/widget.js

```diff
@@ -783,7 +784,7 @@ const resetName = (newName, state) => {
       ? camelCase(newName)
       : newName;
   } else {
-    state.ghostName = config.nameStrings.noNameString;
+    state.ghostName = "";
```

Comment:
Why set the ghost name to blank?

The purpose of the noNameString is for when there is no ghost name set.

If we want to allow there to be a blank name, we'd need to change the way that it is set in the config instead.

Reply:
If the ghost name is set to the nonamestring here, then the widget believes that the ghost’s name is nonamestring (let’s take for an example “Glitched Mythos is Awesome” as the nonamestring).

When you reset, the name will update properly to “Name: Glitched Mythos is Awesome” and using !gn will work as intended. However, if I use a stream deck with !gfn Robert, the widget says the name is “Glitched Mythos is Awesome” and will change the ghost name to “Robert Mythos is Awesome” instead of just “Robert”.

The updatenameDOM takes into account if the state.ghostname is blank or not. If it is blank, it outputs nonamestring. If it has something, it outputs the state. Therefore, the code already took this into account.

Posting from mobile, so I hope this makes sense without proper function capitalization.